components/generic/RelatedChip.vue

```diff
@@ -9,7 +9,7 @@
   >
     <b-img
       v-if="img"
-      :src="img"
+      :src="`${img}&size=w200`"
```

Comment:
Don't do this here. Move it up to the parent component's prop provision.

Reply:
ok